azure-mgmt-resources/src/main/java/com/azure/management/resources/implementation/DeploymentsImpl.java

```diff
@@ -40,8 +41,7 @@
 
     @Override
     public PagedIterable<Deployment> listByResourceGroup(String groupName) {
-        // FIXME: filter & top parameter
-        return this.manager().inner().deployments().listByResourceGroup(groupName, null, null).mapPage(inner -> createFluentModel(inner));
+        return this.manager().inner().deployments().listByResourceGroup(groupName).mapPage(inner -> createFluentModel(inner));
```

Comment:
just confirm, this is change from generator

Reply:
Actually this is Impl change.

The resources RP is among the 1st to migrate. Hence there remains quite some TODO/FIXME.

The method call is actually same, but older generate does not generate the method with default parameters (hence the 2 `null`s in code before).